templates/partials/question-definition.html

```diff
@@ -12,11 +12,12 @@
       },
       "button": {
         "close": _("Hide this"),
+        "contextSuffix": _("content"),
```

Comment:
- This doesn't give enough context for a translator to translate. We can't do string concatenation in this way if it needs to be translated.
- This isn't related to this pull request, another pr should be raised for these changes.

Reply:
As agreed I'll leave this in as it is needed for a breaking change to the design system but it'll be no longer translated. Then we will fix this in a further change to the design system.